c/src/ml-api-service-agent-client.c

```diff
@@ -519,6 +519,29 @@ ml_service_get_pipeline_state (ml_service_h h, ml_pipeline_state_e * state)
   return ret;
 }
 
+/**
+ * @brief Internal function to release ml-service pipeline data.
+ */
+int
+ml_service_pipeline_release_internal (void *priv)
+{
+  _ml_service_server_s *server = (_ml_service_server_s *) priv;
+  g_autoptr (GError) err = NULL;
+  int ret;
+
+  ret = ml_agent_pipeline_destroy (server->id, &err);
+  if (ret < 0) {
+    _ml_error_report_return (ret,
```

Comment:
not `_ml_error_report()` but `_ml_error_report_return`?

Reply:
it is internal error case when failed to release pipeline from ml-agent. (previous code also returns error)